### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by fjphhxx** [Loop3D/LoopStructural#158]
I'm having some problems building faults based on the Modelling Faults piece inside Examples. First I draw two fault lines using QGIS as in the first image. 
![pic1](https://github.com/Loop3D/LoopStructural/assets/69345724/01e87652-23be-42ac-8970-8960eeceb04c)

Then the code runtime reads this shp file and generates the second and third images. 
![pic2](https://github.com/Loop3D/LoopStructural/assets/69345724/de3b30e0-7b93-4428-94a3-08b3f542539a)
![pic3](https://github.com/Loop3D/LoopStructural/assets/69345724/f53f5dc6-4f69-41f2-a2b0-965b51404869)

But this fault generated is very strange. The fourth image is my data. 
![pic4](https://github.com/Loop3D/LoopStructural/assets/69345724/1ae151df-ad7f-4c06-8859-962234a86bce)

Please explain me my error. Thanks.

**Comment by lachlangrose** [comment]
Hi,

Can you provide any code or data so I can recreate the model? I don't know
what you have tried to do.

Thanks

**Comment by fjphhxx** [comment]
Hi, thanks for being able to answer my question. Here are some of my confusions:
I have used your code to build the stratigraphic model I need, but I am having some problems building the fault model.

Inside the attachment is the fault model I want to model and the shp file I used above. I want to use loopstructural to build this fault model.

I would like to ask for advice on what I should do now to model this fault implicitly.
Thanks.
[fault.zip](https://github.com/Loop3D/LoopStructural/files/12445282/fault.zip)
